compat/src/render.js

```diff
@@ -130,6 +130,12 @@ options.vnode = vnode => {
 				delete props.value;
 			}
 
+			// Normalize textarea by setting children to value
+			if (props.value != null && type === 'textarea') {
```

Comment:
Would this handle `defaultValue` too? [React Docs](https://reactjs.org/docs/uncontrolled-components.html#default-values)

Reply:
It does! This runs after the transformation that renames `props.defaultValue` to `props.value`, so it would apply in that case as well. It does make sense to test that case separately though, so I've gone ahead and added one.